strawberry/types/context_wrapper.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+from typing import Any, Dict, Optional
+
+
+@dataclass
+class ContextWrapper:
+    context: Optional[Any]
+    extensions: Optional[Dict[str, Any]]
```

Comment:
Same suggestion as above

```suggestion
    extensions: Optional[Dict[str, Any]] = None
```